src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -679,6 +679,12 @@ public virtual bool CanDetach()
         /// <returns>[Required] Task to track when this is complete</returns>
         abstract public Task EnableTargetAsyncOption();
 
+        /// <summary>
+        /// Obtains a Results object containing the set of features supported by the debugger.
+        /// </summary>
+        /// <returns>Task that will contain the Results. The Results object is optional.</returns>
```

Comment:
Sorry for the confusion - now that you are returning an empty results instead of `null`, you don't need the 'optional' comment anymore. How about - `[Required] Task that will contain the Results.`